extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -113,7 +113,7 @@ class AmpInstagram extends AMP.BaseElement {
     image.setAttribute('layout', 'fill');
     image.setAttribute('referrerpolicy', 'origin');
 
-    this.propagateAttributes(['alt'], image);
+    image.setAttribute('alt', 'Instagram');
```

Comment:
Hi @lisaychuang 
Let's do `'Instagram: ' +	this.element.getAttribute('alt')` here so that page authors can also provide additional `alt`.